src/components/pages/courses/membership-dialog-button.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import Link from 'next/link'
+import FeaturesList from '../../pro-member-features'
+import DialogButton from './dialog-button'
+
+const MembershipDialogButton = ({title, children, buttonText}: any) => {
+  return (
+    <DialogButton
+      buttonText={buttonText}
+      title={title}
+      buttonStyles="text-gray-600 dark:text-gray-300 flex flex-row items-center rounded hover:bg-gray-100 
+        dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out opacity-90 shadow-sm"
+    >
+      <p className="max-w-10 text-center text-gray-700">{children}</p>
+      <p className="max-w-10 text-center text-gray-700">PRO members get:</p>
+      <FeaturesList />
```

Comment:
this is more "robust" than the bullets that were here previously and I'd like to see some screenshots